assets/templates/search/results.tmpl

```diff
@@ -1,12 +1,24 @@
 {{ $lang := .Language }}
+{{ $currentPage := .Data.Pagination.CurrentPage }}
+{{ $itemsPerPage := .Data.Pagination.Limit }}
 <div id="results" class="results">    
     {{ if ne .Data.Response.Count 0 }}
         <div class="search-results ">
             <ul class="list--neutral flush">
-                {{ range .Data.Response.Items }}
+                {{ range $i, $item := .Data.Response.Items }}
+                    {{ $currentPosition := add $i 1 }}
+                    {{ $totalSearchPosition := multiply (subtract $currentPage 1) $itemsPerPage }}
```

Comment:
if $currentPage starts of as `0`
`{{ $totalSearchPosition := multiply (subtract $currentPage 1) $itemsPerPage }}`
Would result in $totalSearchPosition being `-1*$itemsPerPage` Beign negative I assume .Data.Pagination.CurrentPage is always `>1` ?

This line probably should be bumped to be above line 8 somewhere (maybe even just after line 3, as it isn't using and variables declared within the range statement but is being redeclared in the rage statement numerous times and each time yielding the same value.

Reply:
Yup `currentPage` is always (or should always, and if not we probably have bigger issues)`>0`. 

Good spot. Think it was originally using something in the loop but isn't any more. Moved code out of loop.